src/components/Search.jsx

```diff
@@ -1,5 +1,29 @@
-const Search = () => {
-  return <div>Search</div>;
+"use client";
+import { BiSearch } from "react-icons/bi";
+
+const Search = ({ value, onChange, placeholder, classes }) => {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const searchText = e.target[0].search.value;
+    console.log(`Searched: ${searchText}`);
+  };
+
+  return (
+    <div
+      className={`m-[5%] flex flex-row bg-transparent rounded-xl border-2 text-base ${classes}`}
+    >
+      <BiSearch className="mt-[10px] ml-3 text-lg text-code-white fixed" />
```

Comment:
what do u need fixed for?

Reply:
i changed it to absolute so it's relative to the parent, but it's there to not displace the input field. When I take it out, the focus-border of the input field doesn't cover the whole div border which ruins the look of the component. Should I keep the absolute, or should I just remove the focus-border of the input field?